# A high-resolution protein architecture of the budding yeast genome

## Matthew J. Rossi, Prashant K. Kuntala, William K.M. Lai, Naomi Yamada, Nitika Badjatia, Chitvan Mittal, Guray Kuzu, Kylie Bocklund, Nina P. Farrell, Thomas R. Blanda, Joshua D. Mairose, Ann V. Basting, Katelyn S. Mistretta, David J. Rocco, Emily S. Perkinson, Gretta D. Kellogg, Shaun Mahony, B. Franklin Pugh

*Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, Pennsylvania 16802, USA*

*Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York, 14853, USA*

### Correspondence: fp265@cornell.edu

### [www.yeastepigenome.org](http://www.yeastepigenome.org/)
### PMID : [33692541](https://pubmed.ncbi.nlm.nih.gov/33692541/)
### GEO ID : [GSE147927](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147927)

## Overview
The genome-wide protein architecture of chromatin that maintains chromosome integrity and gene regulation is unknown. Here we use ChIP-exo/seq to define this structure in Saccharomyces. We identified 21 ensembles consisting of ~400 different proteins related to DNA replication, centromeres, subtelomeric regions, transposons, and RNA polymerase (Pol) I, II, and III transcription units. DNA replication proteins engulfed a nucleosome, centromeres lacked a nucleosome, and repressive proteins encompassed three nucleosomes at subtelomeric X-elements. We find that most Pol II promoters evolved to lack a regulatory region, having only a core promoter. These constitutive promoters comprised a short nucleosome-free region (NFR) adjacent to a +1 nucleosome, which together bound TFIID to form a pre-initiation complex (PIC). Positioned insulators protected core promoters from upstream events. A small fraction were architected for inducibility, wherein sequence-specific transcription factors (TFs) bound upstream in each promoter to create a nucleosome-depletable regulatory (NDR) region that is distinct from NFRs. We describe TF structural interactions with the genome and cognate cofactors, including nucleosomal and transcriptional regulators RPD3-L, SAGA, NuA4, Tup1, Mediator, and SWI-SNF. Surprisingly, we do not detect TF-TFIID interactions, suggesting that they do stably occur. Thus, our model for gene induction involves TFs, cofactors, and TBP but not TFIID, whereas constitutive transcription involves TFIID but not TFs and cofactors. From this we define a highly integrated network of TF-regulated transcription.

## Downloading processed data
Individual datasets can be browsed and dowloaded via [www.yeastepigenome.org](http://www.yeastepigenome.org/). If you are interested in bulk-downloading all datasets at one time, use [ftp://data1.commons.psu.edu/pub/commons/eberly/pughlab/yeast-epigenome-project](ftp://data1.commons.psu.edu/pub/commons/eberly/pughlab/yeast-epigenome-project) and the [sample key](02_References_and_Features_Files/sample-key.tab).
